src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/DataSource.scala

```diff
@@ -60,7 +94,10 @@ class DataSource(graphFactory: OrientGraphFactory) extends LazyLogging {
 }
 
 class RawlsTransaction(graph: OrientGraph, dataSource: DataSource) {
-  def withGraph[T](f: Graph => T) = f(graph)
+  def withGraph[T](f: Graph => T) = {
+    graph.makeActive()
```

Comment:
because transactions are spanning threads we need to make sure the graph is active